.idea/runConfigurations/webApp.xml

```diff
@@ -0,0 +1,24 @@
+<component name="ProjectRunConfigurationManager">
```

Comment:
Can we also add a `jsApp` run config? It should run `:sample:jsBrowserRun`. 
This one can be renamed to `wasmJsApp`.

Reply:
I think you forgot to add the jsApp config

Reply:
🤔 If that's the case, then I am confused. From what I read, I thought that Kotlin/Js and Kotlin/wasm are two different targets (both web targets though). Kotlin/Js instructions are [here](https://kotlinlang.org/docs/js-project-setup.html) and Kotlin/wasm instructions [here](https://kotlinlang.org/docs/wasm-get-started.html). Kotlin/Js is considered stable and has it's own task `./gradlew jsRun`, while Kotlin/wasm is still experimental and runs with `./gradlew wasmJsBrowserRun`. Or am I missing something?

Reply:
`Kotlin/Js` uses html to build UI (stable)
`Kotlin/Wasm` uses compose to build UI (alpha)

business logic can be shared between the two 

the most inclusive setup is to support both of them and use kotlin/wasm for UI 

can review this [example](https://github.com/JetBrains/compose-multiplatform/tree/master/examples/imageviewer#run-on-web-via-gradle)

Reply:
I see. So when someone builds a Kotlin/Js project and uses this library (or when running `./gradlew jsRun` on the sample) it will still run the wasm app. That's why `main.js.kt` uses `onWasmReady {}`.

Reply:
yes